### PR TITLE
Mt/generated columns benchmark

### PR DESCRIPTION
**Pull request by matriv**

### CODE REVIEW

File: py/README.rst [hunk 1]
```diff
@@ -13,7 +13,7 @@ Usage
 To install it execute::
 
     $ python3.5 -m venv venv
-    $ venv/bin/python -m pip install -r requirements.txt
+    $ venv/bin/python -m pip install -r py/requirements.txt
```
Comment (mfussenegger) [inline py/README.rst]: not sure about this.. Depends if the readme is read from the root folder or if someone is already inside the py folder while reading it. Both variants are probably confusing?

File: py/specs/generated_columns.toml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+     [[setup.data_files]]
+        target = "generated_columns"
+        source = "data/generated_columns.json"
+
+statements = [
+    "refresh table generated_columns"
+]
+
+[[queries]]
+statement = "select (((((((((((((((((((col_1 + col_2) + col_3) + col_4) + col_5) + col_6) + col_7) + col_8) + col_9) + col_10) + col_11) + col_12) + col_13) + col_14) + col_15) + col_16) + col_17) + col_18) + col_19) + col_20) as sum from generated_columns"
```
Comment (mfussenegger) [inline py/specs/generated_columns.toml]: I think the parentheses could be removed, but I'm also okay with keeping them

**Comment by mfussenegger** [comment]
lgtm

File: py/README.rst (hunk 1 above)
Comment (matriv) [inline py/README.rst]: But since the command is venv/bin/python .... means we are in the  ......../crate-stresstest "root" dir, or no?

Comment (mfussenegger) [inline py/README.rst]: Not necessarily.. the instruction `python3.5 -m venv venv` will create a `venv` folder in the `current` directory

**Pull request merged**